tests/python/relay/test_analysis_pipeline.py

```diff
@@ -0,0 +1,144 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
1. The file name isn't correct. Better to be `test_pipeline_executor.py`. Later you could have another file called `test_pipeline_partition.py` or something like that.
2. Please rewrite this file in pytest style. The current test won't be executed by CI.

Reply:
fixed